Testing scripts

AWbosman opened the pull request:
Tested out the _scripts/create_robustness_dist_autoattack.py_ code and fixed two things. 

1. added a test for when AutoAttack module is not installed, as it is required to run this script and returned a warning. 
2. All of a sudden using verbose in our wrapper gave the following error: 
_TypeError: AutoAttack.__init__() got an unexpected keyword argument 'verbose'_ which ment I had to deal with that.

Aaron99B commented on ada_verona/verification_module/attacks/auto_attack_wrapper.py:
```diff
@@ -46,8 +46,9 @@ def execute(self, model: Module, data: Tensor, target: Tensor, epsilon: float) -
         
         adversary = AutoAttack(
-            model, norm=self.norm, eps=epsilon, version=self.version, device=self.device, verbose=self.verbose
+            model, norm=self.norm, eps=epsilon, version=self.version, device=self.device
```
Then let's go with the github version I think. It seems to be better, and I think we used the same in the LION paper. But we should highlight it then in the readme.

AWbosman replied:
check
this will lead to some changes.

AWbosman commented:
Added autoattack back as a module instead of pyautoattack. 
Also changed the install note in _examples/scipts/create_robustness_dist_autoattack.py_
changed back conftest.py and changed the test
Maybe documentation needs to be adjusted?
